Example/ios/Example/AppDelegate.m

```diff
@@ -82,21 +82,11 @@ -(UIView *)findViewByAccessibilityLabel:(UIView *)parent :(NSString *)accessibil
 }
 
 -(NSArray<UIView *> *)cobrowseRedactedViewsForViewController:(UIViewController *)vc {
-  UIView *devMenu = [self findViewByAccessibilityLabel:self.window:@"React Native Debug Menu"];
-  
-  if (devMenu) {
-    return @[self.window.rootViewController.view, devMenu];
-  }
-
   return @[self.window.rootViewController.view];
 }
 
 -(NSArray<UIView *> *)cobrowseUnredactedViewsForViewController:(UIViewController *)vc {
-  UIView *bundlerConfig = [self findViewByAccessibilityLabel:self.window:@"Configure Bundler"];
```

Comment:
Should we also remove the implementation of `findViewByAccessibilityLabel` above? Or maybe it's used elsewhere?

Reply:
argh yea, done